README.md

```diff
@@ -215,6 +214,29 @@ class YourClass {
 }
 ```
 
+## Registering Metrics
+
+The best way to register a metric is via a `static final` class variable as is common with loggers.
+
+```java
+static final Counter requests = Counter.build()
+   .name("requests_total").help("Total requests.").labelNames("path").register();
+```
+ 
+Using the default registry with variables that are `static` is ideal since registering a metric with the same name 
+is not allowed and the default registry is also itself static. You can think of registering a metric, more like 
+registering a definition (as in the `TYPE` and `HELP` sections). The actual samples are tracked via `Child` objects.
```

Comment:
I know what you mean. The name `Child` also adds to the confusion. I initially had trouble trying to register metrics since I would register 2 metrics with the same name so I could add labels later. (Yes, I realize I was approaching it the wrong way).

Since opening this PR I've had to teach several devs to add metrics and the biggest hurdle for each is actually in the registration. 

Should I try removing the references to the `Child` construct and replace it with 'labelled metrics'.